.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -53,3 +52,6 @@ To add a screenshot, please use one of the following formats and/or methods desc
 
 * https://help.github.com/en/articles/file-attachments-on-issues-and-pull-requests
 *
+
```

Comment:
At a high level, "Will you be able to contribute a pull request for this issue?" strikes me as somewhat strong.

What if they say "yes" but it doesn't work out? Will they feel bad?

Instead should we say something softer like "Are you thinking about creating a pull request for this issue?"

That way they won't feel as "on the hook", I would hope.

Reply:
Sure, it does sound less intimidating